lib/AST/ASTPrinter.cpp

```diff
@@ -2357,7 +2357,21 @@ void PrintAST::visitTopLevelCodeDecl(TopLevelCodeDecl *decl) {
 }
 
 void PrintAST::visitIfConfigDecl(IfConfigDecl *ICD) {
-  // FIXME: Pretty print #if decls
+  if (!Options.PrintIfConfig)
+    return;
+
+  for (auto &Clause : ICD->getClauses()) {
+    if (&Clause == &*ICD->getClauses().begin())
+      Printer << tok::pound_if << " "; // FIXME: print condition
```

Comment:
I think this is just a *unimplemented* feature.
For now, added a commit for printing `/* condition */` here.